Updated from stable to 2.0.85 and got a packing issue not finding RID

I also can't get the message in english. I don't know how to force this.... I hate translated error messages.

```
_SdkIncludeDefaultProjectBuildOutputInPackImpl cible) ->
  C:\...msbuild.sdk.extras\2.0.85\Build\RIDs.targets(217,5): error MSB4096: 
L'élément "..\..\..\LICENSE" de la liste d'éléments "None" ne définit aucune valeur pour les métadonnées "Rid".  Pour utiliser ces métadonnées, qualifiez-les en spécifiant %(None.Rid) ou assurez-vous que tous les éléments de cette liste définissent une valeur pour ces métadonnées.
```

source csproj:

```
  <PropertyGroup>
     <PackageLicenseFile>LICENSE</PackageLicenseFile>
 ....
  <ItemGroup>
    <None Include="..\..\..\LICENSE" Pack="true" PackagePath="" />
    ...
```

Any clue ?

--- a/Source/MSBuild.Sdk.Extras/Build/RIDs.targets
+++ b/Source/MSBuild.Sdk.Extras/Build/RIDs.targets
@@ -215,7 +215,7 @@
 
     <!-- Include the runtimes files -->
     <ItemGroup>
-      <None Include="@(RidSpecificOutput->'%(Identity)')" PackagePath="runtimes/%(Rid)/lib/%(TargetFramework)" Pack="true" KeepMetadata="$(ExtrasNoneRidSpecificOutputKeepMetadata)" />
+      <None Include="@(RidSpecificOutput->'%(Identity)')" PackagePath="runtimes/%(RidSpecificOutput.Rid)/lib/%(RidSpecificOutput.TargetFramework)" Pack="true" KeepMetadata="$(ExtrasNoneRidSpecificOutputKeepMetadata)" />
     </ItemGroup>
 
   </Target>

